src/components/filter/DropdownFilter.tsx

```diff
@@ -28,10 +25,13 @@ export const DropdownFilter = ({
   setFilters
 }: Props) => {
   const initiallySelectedItem = { id: 0, index: 0, value: placeholder || '', selected: true };
-  const [dropdownData, setDropdownData] = useState<DropdownProps[]>([
-    initiallySelectedItem,
-    ...data
-  ]);
+  const [dropdownData, setDropdownData] = useState<DropdownProps[]>(() => {
+    if (data[0]?.value === '- Alle -') {
+      return [...data];
+    }
+
+    return [initiallySelectedItem, ...data];
+  });
```

Comment:
why is this change necessary?

Reply:
Because there is no placeholder data in the dropdown array we created for Sue, so we had to add it here. But the array we created for resourceFilter contains the placeholder data. So we needed an extra control.

Reply:
i cannot understand it. what has this code to do with `sue`? can you please give examples for `data` and `initiallySelectedItem` in different cases.

Reply:
this part of the code has been reverted